docs/development/qubes_staging.rst

```diff
@@ -58,7 +58,7 @@ In ``dom0``:
 You may need to edit the filepath above if you downloaded the ISO to a
 different location within the ``sd-dev`` VM. Choose **Install Ubuntu**.
 For the most part, the install process matches the
-:ref:`hardware install flow <install_trusty>`, with a few exceptions:
+:ref:`hardware install flow <install_ubuntu>`, with a few exceptions:
 
   -  Server IP address: use value returned by ``qvm-prefs sd-trusty-base ip``, with ``/24`` netmask suffix
```

Comment:
Should we still keep all the vm names as *trusty? I think we should update those too.

Reply:
so this PR doesn't instruct people to use Xenial for the Qubes staging VMs, that's tracked as a task in #4155